include/bluetooth/nrf/host_extensions.h

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/**
+ * @file
+ * @defgroup bt_nrf_host_extensions Bluetooth vendor specific APIs
+ * @{
+ * @brief Bluetooth Host APIs for Nodic-LL vendor-specific commands.
+ */
+
+#ifndef BT_NRF_HOST_EXTENSIONS_H_
+#define BT_NRF_HOST_EXTENSIONS_H_
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#define BT_HCI_OP_SET_REMOTE_TX_POWER	BT_OP(BT_OGF_VS, 0x010A)
```

Comment:
Do these HCI commands need to be exposed to the application or can they be moved to the .c file or an internal header?